### ISSUE TITLE
Can't find variable: axios error thrown on older browser (since v.1.6.8)

### ISSUE THREAD

**Issue by kamilrychta** [axios/axios#6306]
### Describe the bug

**Axios version 1.6.8**
Axios library set on Chromium version 38 and lower (with help of CDN https://cdn.jsdelivr.net/npm/axios@1.6.8/dist/axios.min.js)

Since your latest Axios upgrade (from 1.6.7 to 1.6.8) we are receiving an error 'Can't find variable: axios' when we try to call an 'axios' method inside the browser console

It works perfectly fine on previous version - v1.6.7

### To Reproduce

To reproduce it, we need to set the script inside the browser console with chrome/chromium version lower or equal to 38.x

Here is a script example:
```
var axiosScript = document.createElement('script');
axiosScript.type = 'text/javascript';
axiosScript.src = 'https://cdn.jsdelivr.net/npm/axios@1.6.8/dist/axios.min.js';  
document.getElementsByTagName('head')[0].appendChild(axiosScript);
```

### Axios Version

1.6.8

### Browser

Chromium 38 and lower

### Browser Version

38

**Comment by justindhillon** [comment]
### Browser and system used
I used chromium version: 38.0.2123.0 which can be found here:
https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/289395/

And I used firefox version 123.0.1. 

On fedora 39.

### Test 1
```js
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
</head>
<body>

<script>
var axiosScript = document.createElement('script');
axiosScript.type = 'text/javascript';
axiosScript.src = 'https://cdn.jsdelivr.net/npm/axios@1.6.8/dist/axios.min.js';  
document.getElementsByTagName('head')[0].appendChild(axiosScript);
</script>

</body>
</html>
```

This does not work on chromium 38, but does work on firefox 123.

### Test 2
```html
<!DOCTYPE html>
<html>
<body>
<script src="https://cdn.jsdelivr.net/npm/axios@1.6.8/dist/axios.min.js"></script>
<script>
  // Make a GET request to a URL
  axios.get('https://jsonplaceholder.typicode.com/posts/1')
    .then(function (response) {
      // Handle success
      console.log('Response:', response.data);
    })
    .catch(function (error) {
      // Handle error
      console.error('Error:', error);
    });
</script>
</body>
</html>
```

This does not work on chromium 38, but does work on firefox 123.

### Conclusion
Although neither of these code samples work with chromium 38, they also don't work with any version of axios. It seems as though axios has never worked with chromium 38.

**Comment by mofipp** [comment]
I am using version v1.7.2 and the same error occurs in the mobile QQ browser.

Here's an example，Use mobile QQ browser to open the problem can be repeated.

https://codepen.io/mofipp/pen/gOJLawg

**Comment by jasonsaayman** [comment]
i dont think we can support such a old version on the v1x branch, maybe looking into v0x will solve this
